cmd/node-installer/uninstall.go

```diff
@@ -45,6 +46,16 @@ var uninstallCmd = &cobra.Command{
 
 		config.Runtime.ConfigPath = distro.ConfigPath
 
+		optionsJson := os.Getenv("RUNTIME_OPTIONS")
```

Comment:
Could probably use a shared helper for this one (between install and uninstall)?